src/Raven.Studio/typescript/commands/database/documents/getDocumentsPreviewCommand.ts

```diff
@@ -28,26 +28,38 @@ class getDocumentsPreviewCommand extends commandBase {
     static readonly ArrayStubsKey = "$a";
     static readonly TrimmedValueKey = "$t";
 
-    constructor(private database: database, private skip: number, private take: number, private collectionName: string, private previewBindings?: string[], private fullBindings?: string[]) {
+    constructor(private database: database, private skip: number, private take: number, private collectionName: string,
+                private previewBindings?: string[], private fullBindings?: string[], private continuationToken?: string) {
         super();
     }
 
     execute(): JQueryPromise<pagedResultWithAvailableColumns<document>> {
-        const args = {
+        
+        const argsWithContinuationToken = {
+            collection: this.collectionName,
+            binding: this.previewBindings,
+            fullBinding: this.fullBindings,
+            "continuation-token": this.continuationToken
```

Comment:
@karmeli87  Don't we use camelCase for url arguments?

Reply:
@Danielle9897 as agreed with marcin please change this to camelCase and verify it is working as expected.

You should also modify on the server side this line:
`public const string ContinuationTokenQueryString = "continuation-token";`

Reply:
done